cmd/main.go

```diff
@@ -135,9 +140,9 @@ func prepareRestAndRPCServer(log *slog.Logger, cfg config.Config, rpcProxyHandle
 			w.WriteHeader(http.StatusServiceUnavailable)
 		}
 	}))
-	m.Handle("/rpc", rpcProxyHandler)
-	m.Handle("/rest", restProxyHandler)
-	m.Handle("/", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	m.Handle("/rpc/", rpcProxyHandler)
```

Comment:
not sure trailing slashes should be here

Reply:
The trailing slash here is to allow for the `ServerMux` of the rpc & rest proxy to be able to get all traffic prefixed by those (`/rest/*` and `/rpc/*`), otherwise traffic was not being routed as the previous routes were specifically only for `/rest` and `/rpc` which makes me wonder if this ever worked before...

Reply:
oh, intetesting